ts/parsers/paragraphparser.ts

```diff
@@ -1,5 +1,5 @@
 import { getValueAtPath } from "../helpers";
```

Comment:
just noticed it now but do these files need to be suffixed with 'parser'? 

`ts/parsers/paragraph.ts` will suffice no? 

At the very least, I'd expect camelcasing, `ts/parsers/paragraphParser.ts`

Reply:
They were named originally like that, didn't change, however the scripts i created i did made them camel case, going to change them.